### PR TITLE
Nominate JenTing Hsiao for core maintainer

### PR DESCRIPTION
**Pull request by carlisia**
All @vmware-tanzu/velero-maintainers: please add your vote 👍 👎 to promote JenTing Hsiao to a core maintainer, thank you.

Signed-off-by: Carlisia <carlisia@vmware.com>

### CODE REVIEW

**Comment by carlisia** [comment]
c/c @jenting.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 | Ashish Amarnath | [ashish-amarnath](https://github.com/ashish-amarnath) | [VMware](https://www.github.com/vmware/) |
 | Bridget McErlean | [zubron](https://github.com/zubron) | [VMware](https://www.github.com/vmware/) |
 | Dave Smith-Uchida | [dsu-igeek](https://github.com/dsu-igeek) | [VMware](https://www.github.com/vmware/) |
+| JenTing Hsiao | [jenting](https://github.com/jenting) | [jenting](https://github.com/SUSE/)
```
Comment (zubron) [inline MAINTAINERS.md:14]: The text for the second link should be SUSE.

Comment (carlisia) [reply MAINTAINERS.md:14]: argh...... :)

**Comment by nrb** [comment]
Per our [governance rules](https://github.com/vmware-tanzu/velero/blob/main/GOVERNANCE.md), we'll need 4 votes as we currently have 5 maintainers.

I'll take Carlisia's nomination as one vote, so that leaves 3 approvals on the PR.

**Review by nrb — APPROVED** [review]
Thanks very much for your contributions @jenting!

**Comment by ashish-amarnath** [comment]
👍🎉🥳

**Comment by carlisia** [comment]
We have quorum! 🎉

**Review by zubron — APPROVED** [review]
:partying_face:

**Pull request merged**